bin/deku_node.re

```diff
@@ -219,9 +219,7 @@ let node = {
       current_validators
       |> List.mapi((i, validator) => {
            (
-             switch (validator) {
```

Comment:
Nit: this change, and another one similar to this later on, looks unrelated to the PR's objectives.